assets/tests/hooks/usePersistedStateReducer.test.ts

```diff
@@ -86,30 +90,28 @@ describe("usePersistedStateReducer", () => {
 
     expect(state.selectedVehicleId).toEqual("vehicle_id")
 
-    // first call is persisting the initial state
-    // second call is persisting the edit we're testing
-    expect(window.localStorage.setItem).toHaveBeenCalledTimes(2)
-    const persistedState = JSON.parse(
-      (window.localStorage.setItem as jest.Mock).mock.calls[1][1]
-    )
+    // last call is persisting the edit we're testing
+    const calls = (window.localStorage.setItem as jest.Mock).mock.calls
+    const lastCallIndex = calls.length - 1
+    const persistedState = JSON.parse(calls[lastCallIndex][1])
     expect(persistedState.selectedVehicleId).toEqual("vehicle_id")
   })
 
   test("loads settings from the backend", () => {
     ;(appData as jest.Mock).mockImplementationOnce(() => ({
-      settings: JSON.stringify({
+      userSettings: JSON.stringify({
         ladder_page_vehicle_label: "run_id",
         shuttle_page_vehicle_label: "run_id",
       }),
     }))
     const { result } = renderHook(() => usePersistedStateReducer())
     const [state] = result.current
-    expect(state.settings.shuttleVehicleLabel).toEqual(
+    expect(state.userSettings.shuttleVehicleLabel).toEqual(
       VehicleLabelSetting.RunNumber
     )
   })
 
-  test("if settings are in localstorage, copies them to the backend and uses them", () => {
+  test("if user settings are in localstorage, copies them to the backend and uses them", () => {
```

Comment:
This test should also test that the route settings are copied from local storage to the backend.